libs/controllers/config/NodeConfigData.py

```diff
@@ -61,6 +64,16 @@ def deserialize(bits):
         """ deserialize the node config """
         return NodeConfigData(**umsgpack.loads(bits))
 
+    def add_neighbour(self, addr, node_config):
+        print("adding neighbour")
+        self.neighbours[addr] = node_config
+        print(self.neighbours)
+
+    def remove_neighbour(self, addr):
+        print("removing neighbour")
+        self.neighbours.pop(addr)
```

Comment:
First off, not the place for this.

Second off, we have a logger now, please use that instead.